opentreemap/treemap/templatetags/form_extras.py

```diff
@@ -464,6 +468,27 @@ def resolve_label_and_identifier(self, context):
 
         return label, identifier
 
+    def _fill_in_typeahead(self, field, field_name, model):
+        relation_lookup_infos = {
+            'treemap.models.User': {
+                'url': reverse(
+                    'users', kwargs={
+                        'instance_url_name': model.instance.url_name
+                    }
+                ) + '?q=%Q%',
+                'placeholder': _('Please type a user-name'),
```

Comment:
This should be `username` and not `user-name` to match what we have used elsewhere in the site